packages/banxa/package.json

```diff
@@ -0,0 +1,209 @@
+{
+  "name": "@yoroi/banxa",
+  "version": "1.0.0",
+  "description": "The Banxa integration package of Yoroi SDK",
+  "keywords": [
+    "yoroi",
+    "cardano",
+    "banxa",
+    "browser",
+    "react",
+    "react-native"
+  ],
+  "homepage": "https://github.com/Emurgo/yoroi/packages/banxa#readme",
+  "bugs": {
+    "url": "https://github.com/Emurgo/yoroi/issues"
+  },
+  "repository": {
+    "type": "github",
+    "url": "https://github.com/Emurgo/yoroi.git",
+    "directory": "packages/banxa"
+  },
+  "license": "Apache-2.0",
+  "author": "EMURGO Fintech <support@emurgo.com> (https://github.com/Emurgo/yoroi)",
+  "contributors": [
+    {
+      "name": "Juliano Lazzarotto",
+      "email": "30806844+stackchain@users.noreply.github.com"
+    }
+  ],
+  "main": "lib/commonjs/index",
```

Comment:
A local build is needed to be able to use the package

Reply:
yap, doing that for all now, cuz it needs to resolve the package dependency order, I've added lerna that does so. There is a new script on the root that bootstrap everything and watch for changes, so while working on the packages it will watch and rebuild pkgs in the right order.